data/USDC/data.json

```diff
@@ -5,40 +5,25 @@
   "description": "USDC is a faster, safer, and more efficient way to send, spend, and exchange money around the globe.",
```

Comment:
can we get quick modification to this description about this being native usdc minted by Circle's CCTP protocol?